operator/cmd/configurator/bootstrap.go

```diff
@@ -0,0 +1,36 @@
+package configurator
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/redpanda-data/redpanda-operator/operator/pkg/clusterconfiguration"
+)
+
+// Template out the bootstrap file
+// This takes an input template, resolves any remaining external references, then writes out the resulting bootstrap file
+func templateBootstrapYaml(inFile, outFile string) error {
+	var template map[string]clusterconfiguration.ClusterConfigTemplateValue
+	buf, err := os.ReadFile(inFile)
+	if err != nil {
+		return fmt.Errorf("cannot load bootstrap template file: %w", err)
+	}
+	if err := json.Unmarshal(buf, &template); err != nil {
+		return fmt.Errorf("cannot parse bootstrap template file: %w", err)
+	}
+
+	var config []string
+	for k, v := range template {
```

Comment:
I don't think this will ever matter but do you think we should be concerned that the key ordering could get shuffled around here?

Reply:
I think probably not, the yaml input is an object. If there's some implicit ordering we can fix that up. The result is only visible to the local process.